gateway/models.py

```diff
@@ -113,10 +111,9 @@ def serialize(self):
             }
         if self.ble_extension:
             response.update(self.ble_extension.serialize())
```

Comment:
Why is the ble extension not in scim_ext_read and handled separately like this?

Reply:
I just haven't gotten there yet.